src/main/python/mind_your_ps_and_qs.py

```diff
@@ -1,16 +1,20 @@
 import requests
 
 
-def get_ps_and_qs(ps_and_qs, factors, idx, curr_factor_one, curr_factor_two):
-    if idx == len(factors):
-        if curr_factor_one == 1 or curr_factor_two == 1:
-            return
+def get_ps_and_qs(coprimes):
+    ps_and_qs = []
+    get_ps_and_qs_helper(ps_and_qs, coprimes, 0, 1, 1)
+    return ps_and_qs
+
+
+def get_ps_and_qs_helper(ps_and_qs, coprimes, idx, curr_factor_one, curr_factor_two):
+    if idx == len(coprimes):
         ps_and_qs.append({curr_factor_one, curr_factor_two})
```

Comment:
should append a tuple instead of a set. So it should be `ps_and_qs.append((curr_factor_one, curr_factor_two))`

Reply:
agree